packages/zeebe-element-templates-json-schema/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [@camunda/zeebe-element-templates-json-schema](https://gi
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.17.0
+
+* `FIX`: disallow `bpmn:Message#zeebe:subscription#property` binding for `bpmn:SendTask`
+
+### Breaking Changes
```

Comment:
Not sure if it is breaking, as it is a bug fix.

Reply:
OK let's change it.